controls.py

```diff
@@ -0,0 +1,110 @@
+class control():
```

Comment:
Style recommendation: consider using `UpperCamelCase` for class names.

Also, the `control` class is never used in any other file. Need to import it somewhere.